test/test_nodes.py

```diff
@@ -24,24 +24,21 @@
 import numpy as np
 import numpy.linalg as la
 import pytest
+import modepy.shapes as shp
+import modepy.nodes as nd
 
 
 @pytest.mark.parametrize("dims", [1, 2, 3])
 def test_barycentric_coordinate_map(dims):
-    from random import Random
-    rng = Random(17)
-
-    n = 5
-    unit = np.empty((dims, n))
+    n = 100
     from modepy.tools import (
-            pick_random_simplex_unit_coordinate,
             unit_to_barycentric,
             barycentric_to_unit,
             barycentric_to_equilateral,
             equilateral_to_unit,)
 
-    for i in range(n):
-        unit[:, i] = pick_random_simplex_unit_coordinate(rng, dims)
+    rng = np.random.Generator(np.random.PCG64(17))
```

Comment:
Why the weird randon number generator?

Reply:
Just copied their example:

https://numpy.org/doc/stable/reference/random/generator.html

Reply:
TIL I thought the Mersenne twister was the default. Cool!